assets/src/edit-story/components/checklist/checklist.js

```diff
@@ -60,6 +62,53 @@ const Wrapper = styled.div`
 // TODO make this responsive so that title bar is never covered by popup.
 const StyledNavigationWrapper = styled(NavigationWrapper)``;
 
+const ThroughputPopup = forwardRef(function ThroughputPopup(
+  { isOpen, children, close },
+  ref
+) {
+  const [isMounted, setIsMounted] = useState(false);
+  return (
+    <Popup
+      popupId={POPUP_ID}
+      isOpen={isOpen}
+      ariaLabel={CHECKLIST_TITLE}
+      shouldKeepMounted
+      onEnter={() => setIsMounted(true)}
+      onExited={() => setIsMounted(false)}
+    >
+      <mountedContext.Provider value={isMounted}>
+        {isMounted ? (
```

Comment:
Naive question: why can't we use `isOpen` here?

Reply:
not naive at all. that was my original approach, but it causes children to get unmounted while the exit animation is still playing which results in some unwanted layout and nasty transition.

using `isMounted` , which looks at the transition callbacks, allows this to wait until the exit animation is complete before removing the children components

Reply:
Thanks! Any reason why we should create a new context vs. adding `isMounted` to `useChecklist`?

Reply:
nope. good call out. happy to move it in there to consolidate it a bit